app/views/partials/foyer/ressources/types.html

```diff
@@ -1,16 +1,15 @@
 <h1>{{ pageTitle }}</h1>
 
 <p>
-  Sélectionnez les ressources perçues <strong>entre {{ debutAnneeGlissante }} et {{ finAnneeGlissante }}</strong>,
+  Sélectionnez les ressources perçues <strong>depuis {{ debutAnneeGlissante }}</strong>,
   vous pourrez ensuite saisir les montants.
 </p>
 
 <form method="post" name="form" novalidate ng-submit="submit()">
   <div class="ressource-types-block">
     <div class="capture-{{ category.id }}" ng-repeat="category in ressourceCategories">
-      <div class="capture-title">
-        <h2>{{ category.label }}</h2>
-      </div>
+      <details>
+        <summary class="capture-title">{{ category.label }}</summary>
```

Comment:
`<details>` needs to be [polyfilled](https://en.wikipedia.org/wiki/Polyfill) for IE11. If you want to use it, add [this one](https://github.com/MattiSG/Element.details/tree/mattisg).